jobs/sync-stream-metadata.Jenkinsfile

```diff
@@ -29,12 +29,16 @@ cosaPod(configMaps: ["fedora-messaging-cfg"], secrets: ["fedora-messaging-coreos
         // they're different from the checkout, and copy out the new versions
         // if so
         streams.production.each{stream ->
-            for (subdir in ["streams", "updates"]) {
+            shwrap("""
+                python -c 'import sys, yaml, json; json.dump(yaml.safe_load(sys.stdin.read()), sys.stdout)' \
+                    < release-notes/${stream}.yaml > release-notes/${stream}.json
+            """)
+            for (subdir in ["streams", "updates", "release-notes"]) {
```

Comment:
This will overwrite the newly converted JSON files above.

OK this is messy because it interacts with the naive change detection we try to do. I think rather than trying to get change detection right for release notes, I'd say let's just have it always re-upload them.

Let me try tweaking this.